caas/kubernetes/provider/k8s.go

```diff
@@ -1441,6 +1493,23 @@ func (k *kubernetesClient) jujuStatus(podPhase core.PodPhase, terminated bool) s
 	}
 }
 
+func (k *kubernetesClient) jujuWorkloadStatus(podPhase core.PodPhase, terminated bool) status.Status {
```

Comment:
This method is unused and should be deleted

Reply:
Oh shoot, that got lost in my touch up just now, I've re-introduced it so we can translate the Operator status into a juju application workload status.

Reply:
I think we still want to use jujuStatus() at this layer and do any translation (if needed) in the facade.